### PR TITLE
Feat/bonds page

### PR DESCRIPTION
**Pull request by xpn9**

### CODE REVIEW

File: src/common/utils.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { ethers } from 'ethers'
 import { MerkleTree } from 'merkletreejs'
 import getTokenList from 'get-token-list'
 import defaults from './defaults'
+const EST_BLOCKS_PER_HOURS = 262
```
Comment (yellowgh0st) [inline src/common/utils.js:5]: Always use defaults.js for constants.

File: src/common/utils.js [hunk 2]
```diff
@@ -201,8 +202,19 @@ const getMerkleProofForAccount = (account, snapshot) => {
 	return proof
 }
 
+const convertBlocksToDays = (blocks) => {
+	return blocks / (EST_BLOCKS_PER_HOURS * 24)
+}
+
+const getTokenPrice = (tokenId) => {
+	return fetch(`${defaults.api.coingeckoUrl}?vs_currencies=usd&ids=${tokenId}`)
```
Comment (yellowgh0st) [inline src/common/utils.js:210]: What kind of tokens will be priced using this API?

**Review by yellowgh0st — CHANGES_REQUESTED** [review]

**Comment by yellowgh0st** [comment]
@Satoshi-Naoki Please check.

File: src/common/utils.js (hunk 2 above)
Comment (xpn9) [reply src/common/utils.js:210]: Principal & payout tokens. The prices are needed for calculating ROI, treasury balance..

**Pull request closed without merging**